### PR TITLE
refactor(ng-dev): expose git and github client classes in ng-dev entry-point

### PR DESCRIPTION
**Pull request by devversion**
These classes will be needed for interacting with the release versioning APIs.

### CODE REVIEW

File: ng-dev/index.ts [hunk 1]
```diff
@@ -22,6 +22,12 @@ export * from './release/versioning';
 // to the dev-infra log which is stored on failures.
 export * from './utils/console';
 
+// Exposes Git/Github client classes needed for interacting with some of the
+// release versioning APIs.
+export * from './utils/git/authenticated-git-client';
```
Comment (josephperrott) [inline ng-dev/index.ts:27]: Can change this to expose the specific symbols we want rather than all?

**Review by josephperrott — APPROVED** [review]
LGTM

File: ng-dev/index.ts (hunk 1 above)
Comment (devversion) [reply ng-dev/index.ts:27]: Yeah, could do. Just not sure if it's worth maintaining that list. It's already quite unclear what specific APIs e.g. the versioning API transitively requires 😓

Comment (devversion) [reply ng-dev/index.ts:27]: Let me know, happy to go with either approach (star would be consistent with the file curently)

Comment (josephperrott) [reply ng-dev/index.ts:27]: We can leave it as is, I am surprised we aren't able to just provide the specific ones here as for instance `./utils/git/authenticated-git-client` only has two exported symbols.

**Pull request merged**